python/pyproject.toml

```diff
@@ -38,6 +38,8 @@ langchain-core = {version = "^0.3.41", optional = true}
 langchain-community = {version = "^0.3.19", optional = true}
 boto3 = {version = "^1.37.5", optional = true}
 acp-sdk = "^0.0.4"
+cachetools = "^5.5.2"
+types-cachetools = "^5.5.0.20240820"
```

Comment:
This should go to dev dependencies

Reply:
https://github.com/i-am-bee/beeai-framework/pull/602/commits/c4aa9dfa29408bd8af2f50516ba4f6834c4458ca